From 4178a29308288d7b8a5de3634f2a13e54d0e919a Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Wed, 29 Nov 2017 15:33:09 +0800
Subject: [PATCH] ARM: dts: rockchip: rk312x-android: put device cma to NORMAL
 zone

NORMAL zone limits at 0x90000000.

The device cma region which is used by camera should be set to
NORMAL zone to avold memory fragment.

The device cma region info:
[    0.000000] Reserved memory: created CMA memory pool at 0x88000000, size 24 MiB
[    0.000000] Reserved memory: initialized node region@8f000000, compatible id shared-dma-pool

The default cma region info:
[    0.000000] cma: Reserved 16 MiB at 0x9f000000

Change-Id: I8b1a099c5fa3a2d90acf709c4272d88b97e0c5bd
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rk312x-android.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rk312x-android.dtsi b/arch/arm/boot/dts/rk312x-android.dtsi
index 0886b93b3415..e4ed16e0df0f 100644
--- a/arch/arm/boot/dts/rk312x-android.dtsi
+++ b/arch/arm/boot/dts/rk312x-android.dtsi
@@ -123,10 +123,10 @@
 		#size-cells = <1>;
 		ranges;
 
-		cma_region: cma-region {
+		cma_region: region@88000000 {
 			compatible = "shared-dma-pool";
 			reusable;
-			size = <0x1000000>;
+			reg = <0x88000000 0x1800000>;
 		};
 
 		ramoops_mem: ramoops@68000000 {
-- 
2.35.3

